scripts/cmscp.py

```diff
@@ -485,6 +485,10 @@ def perform_local_stageout(local_stageout_mgr, \
         ## Fallback to previous behaviour where phedex is queried for location
         if source_site == 'unknown':
             source_site = G_NODE_MAP.get(dest_temp_se, 'unknown')
+        
+        # If fallback stageout happens, PNN can be different as source
+        if 'PNN' in stageout_info and stageout_info['PNN'] != source_site:
```

Comment:
should I remove it? For sure it does not hurt, but clear that cmscp has to change some variables and use pnn and psns, I think